libs/localization/src/lib/en.json

```diff
@@ -1774,7 +1774,13 @@
         "micro": "Micro"
       },
       "classSelectionDropdown": "Select class(es)",
-      "iouthresholdDropdown": "IoU Threshold",
+      "iouthresholdDropdown": {
+        "name": "IoU Threshold",
+        "description": "The Intersection over Union score is calculated as the ratio of area of intersection to area of union of the ground truth and predicted bounding boxes. A higher IoU score means the predicted box needs to more closely resemble the ground truth bounding box to be deemed a correct prediction.",
```

Comment:
nit: should intersection and union be capitalized ?

Reply:
Kept it as it is as it's mentioned as `area of intersection/union`